pkgs/by-name/xc/xcodes/package.nix

```diff
@@ -1,24 +1,25 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, swift
-, swiftpm
-, swiftpm2nix
-, makeWrapper
-, aria2
+{
+  lib,
+  fetchFromGitHub,
+  swift,
+  swiftpm,
+  swiftpm2nix,
+  makeWrapper,
+  aria2,
 }:
 let
   generated = swiftpm2nix.helpers ./generated;
+  stdenv = swift.stdenv;
```

Comment:
I’m not sure if this is any different from `swiftPackages.stdenv` but I think the latter is canonical.

Reply:
Searched on the repo, and you're right - it seems to be more widely adopted.
Just pushed this change, thanks!